docs/recipes/Code-Splitting-Using-Holocron.md

```diff
@@ -2,6 +2,77 @@
 
 # Code Splitting Using Holocron
 
-Recipe is forthcoming.
+> To view a working sample of code splitting in One App, check out the sample module [Franks Burgers](../../prod-sample/sample-modules/franks-burgers/0.0.0/README.md)
+
+One App uses `one-app-bundler` to support dynamic importing and code splitting, out of the box.
+`one-app-bundler` uses `webpack` and `babel` (with `@babel/plugin-syntax-dynamic-import`) under
+the hood to understand the `import()` syntax and automatically split your module where ever
+the `import()` statement is used within your module. Let's go through a quick example to get
+a better grasp of code splitting.
+
+Let's start with a sample Holocron module project:
+
+```
+root
+|── locale
+|   └── en-US.json
+|── src
+|   ├── chunk.js
+|   └── index.js
+└── package.json
+```
+
+and import our chunk using the dynamic `import()` syntax like so:
+
+`src/index.js`
+```jsx
+import React from 'react';
+
+export default function MyModule() {
```

Comment:
this example needs to be updated to be more aligned with my comment above

Reply:
Example has been updated to use a variety of different examples (cc27a15)